data/gamerules.txt

```diff
@@ -45,7 +45,43 @@ gamerules
 	# ALLOWABLE VALUES: any integer >= 0
 	"npc max mining time" 3600
 	
-	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their health is at or below this fraction.
+	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their
+	# health is at or below this fraction.
 	# DEFAULT: .75 (75 percent)
 	# ALLOWABLE VALUES: any value between 0. and 1.
-	"universal frugal threshold" .75
+	"universal frugal threshold" 0.75
+
+	# Minimum percentage of a ship or outfit value at maximum depreciation. A value of .25 (25 percent) means
+	# a fully depreciated outfit that normally sold at 1000 credits will be sold for 250 credits.
+	# DEFAULT: .25 (25 percent)
+	# ALLOWABLE VALUES: any value between 0. and 1.
+	"depreciation min" 0.25
+
+	# Amount of days before a new outfit and ship begin to depreciate.
+	# DEFAULT: 7 (7 days)
+	# ALLOWABLE VALUES: any integer >= 0.
+	"depreciation grace period" 7
+
+	# First factor of depreciation calculation. The second factor is "depreciation max age" below
+	# This is the Percentage of a ship or outfit value remains per day after the grace period.
+	# The value is then multiplied by previous value on the next day, and repeat.
+	# So this can be calculated by "remaining daily percent" = "daily percent" ^ (age - "grace period")
```

Comment:
What does `"remaining daily percent"` mean?
```suggestion
	# So this can be calculated by "remaining daily proportion" = "daily proportion" ^ (age - "grace period")
```